README.md

```diff
@@ -53,6 +53,8 @@ If there was an error with the build, rebuild using  the  `--no-cache` option li
 
 The site relies primarily on USWDS2 styles pulled in via the [uswds-jekyll](https://github.com/18F/uswds-jekyll) theme.
 
+Please ensure that style updates are consistent with our brand's [colors](https://brand.18f.gov/color-palette/), [typography](https://brand.18f.gov/typography/), and [iconography](https://brand.18f.gov/icons/).
+
 Per this, there are generally two main scss files:
 
 * **_sass/_uswds-theme-theme-settings.scss** This stylesheet pulls in all of the [USWDS2 theme variables](https://designsystem.digital.gov/documentation/developers/#sass-and-theme-settings) that are set in the `_sass/_theme` directory. When updating a style **first check** if it can be updated globally in the theme prior to creating a custom theme.
```

Comment:
I'd replace 'When updating a style **first check** if it can be updated globally in the theme prior to creating a custom theme.' with 'If you want to modify a style, try to make the change globally using the theme and theme variables before creating themes.